loopbackStarter/manifest.yml

```diff
@@ -0,0 +1,10 @@
+applications:
+- path: .
+  memory: 1G
```

Comment:
The app was crashing due to not having enough memory. Since I have truncated the sample data significantly, it might now work with lesser amount. Let me check and update.